config/game_options.txt

```diff
@@ -215,7 +215,7 @@ SECURITY_SCALING_COEFF 8
 ## The number of objectives traitors get.
 ## Not including escaping/hijacking.
 TRAITOR_OBJECTIVES_AMOUNT 2
```

Comment:
Maybe tweak tot objectives to 3. Based on experience. 2 tends towards boring rounds. Specially when we are shooting for 2 hour rounds.

Reply:
redundant